src/Request/AbstractRequest.php

```diff
@@ -77,6 +98,11 @@ public function getParameters()
             $parameters['NUMAPPEL'] = $this->getCallNumber();
         }
 
+        // Direct Plus requests special case.
+        if (null !== $this->getSubscriberRef()) {
+            $parameters['REFABONNE'] = $this->getSubscriberRef();
```

Comment:
Do you plan on replacing the french hardcoded strings ? It would be best to have your own vocabulary and stick to it.